package.json

```diff
@@ -191,48 +188,49 @@
     "eslint-plugin-react": "^7.27.1",
     "eslint-plugin-react-hooks": "^4.3.0",
     "eslint-plugin-testing-library": "^5.0.4",
-    "favicons": "^6.2.2",
-    "favicons-webpack-plugin": "^5.0.2",
-    "fork-ts-checker-webpack-plugin": "^6.4.2",
-    "html-inline-css-webpack-plugin": "^1.11.1",
-    "html-webpack-plugin": "^5.5.0",
-    "husky": "^7.0.0",
+    "favicons": "^7.2.0",
+    "favicons-webpack-plugin": "^6.0.1",
+    "fork-ts-checker-webpack-plugin": "^6.5.3",
+    "html-inline-css-webpack-plugin": "^1.11.2",
+    "html-webpack-plugin": "^5.6.0",
+    "husky": "^7.0.4",
     "i18next-parser": "^5.4.0",
     "is-ci": "^3.0.1",
     "jest-styled-components": "^7.0.8",
-    "jsdom": "^19.0.0",
+    "jsdom": "^24.0.0",
     "lint-staged": "^12.1.2",
-    "mini-css-extract-plugin": "^2.4.5",
+    "mini-css-extract-plugin": "^2.8.1",
     "native-ext-loader": "^2.3.0",
-    "nock": "^13.2.1",
+    "nock": "^14.0.0-beta.5",
     "node-bindings-loader": "^1.5.0",
     "playwright": "^1.29.1",
-    "prettier": "^2.5.1",
+    "prettier": "^2.8.8",
     "range-parser": "^1.2.1",
     "react-dev-utils": "^11.0.4",
     "react-svg-loader": "^3.0.3",
-    "serve": "^13.0.2",
+    "serve": "^13.0.4",
     "snapshot-diff": "0.9.0",
     "tmp-promise": "^3.0.3",
-    "ts-node": "^10.5.0",
-    "tsconfig-paths": "^3.12.0",
-    "tsconfig-paths-webpack-plugin": "^3.5.2",
-    "typescript": "^4.6.2",
-    "vite-tsconfig-paths": "^3.5.0",
-    "vitest": "^0.14.2",
-    "webpack": "^5.64.4",
-    "webpack-bundle-analyzer": "^4.5.0",
-    "webpack-cli": "^4.9.1",
-    "webpack-dev-server": "^4.6.0",
+    "ts-node": "^10.9.2",
+    "tsconfig-paths": "^3.15.0",
+    "tsconfig-paths-webpack-plugin": "^4.1.0",
+    "typescript": "4.9.5",
+    "vite-tsconfig-paths": "^4.3.2",
+    "vitest": "^1.4.0",
+    "webpack": "^5.91.0",
+    "webpack-bundle-analyzer": "^4.10.1",
+    "webpack-cli": "^4.10.0",
+    "webpack-dev-server": "^4.15.2",
     "webpackbar": "^5.0.2"
   },
   "resolutions": {
     "@graphql-codegen/cli@2.3.0": "patch:@graphql-codegen/cli@npm:2.3.0#.yarn/patches/@graphql-codegen-cli-npm-2.3.0-4b6eb70c79",
```

Comment:
Is this needed anymore since you upgraded codegen?

Reply:
*removed*